commands/mute/unmute.js

```diff
@@ -0,0 +1,62 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: 'unmute',
+    description: 'Unmute a user',
+
+    execute(msg, con) {
+        var toUnmuteDisplay = undefined;
+
+        if (!msg.member.roles.cache.some(
+        role => role.name === "Moderator" || role.name === "Admin" || role.name === "Super User")) {
+            return msg.reply("You must be a super user, moderator, or admin to use this command.");
+        } 
+
+        const toUnmute = msg.mentions.members.first();
+        if (!toUnmute) {
+            return msg.reply("Please provide a user to unmute.");
+        } else if (!toUnmute.roles.cache.some(role => role.name === "Muted")) {
+            return msg.reply("This user is already unmuted.");
+        } else {
+            toUnmuteDisplay = toUnmute.displayName;
```

Comment:
Are we doing anything with `toUnmuteDisplay`? Do we need it? If not, let's get rid of it.

Reply:
Done